string NAME=oriented

ObjectDrone d

bool truth_value
